src/openassetio-core-c/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+
+#----------------------------------------------------------------------
+# Public headers
+
+set(public_header_source_root ${CMAKE_CURRENT_LIST_DIR}/include)
+
+# Installation location for install phase.
+# TODO(DF): When CMake 3.23 is released, use FILE_SET.
+install(
+    DIRECTORY
+    ${public_header_source_root}/openassetio
+    DESTINATION
+    ${CMAKE_INSTALL_INCLUDEDIR}
+)
+
+
+#-----------------------------------------------------------------------
+# Create core C target
```

Comment:
[fix] Add ability to disable `C` API builds, on by default.

Reply:
Added a `OPENASSETIO_ENABLE_C` option (enabled by default) in ac2f887e